earth_enterprise/src/server/gehttpd_init

```diff
@@ -95,6 +95,31 @@ if grep -q -s 'Ubuntu' /etc/lsb-release ; then
       RETVAL=$?
     }
 
+elif grep -q -s 'LinuxMint' /etc/lsb-release ; then
```

Comment:
If this is a copy of the Ubuntu block, I'd recommend having a single block and checking for both distributions in the previous block.

Reply:
Ah yeah good point, I will fix it